components/tracer/.env.example

```
@@ -349,6 +349,60 @@ TRACER_TLS_KEY_FILE=
+# plugin-auth) via Consul instead of static addresses. Default is OFF: when
+# SD_ENABLED=false a no-op Manager is wired, no Consul connection is attempted,
+# and Resolve returns the static fallback host unchanged.
+#
+# When SD_ENABLED=true, SD_EXTERNAL_ADDRESS (or SD_INTERNAL_ADDRESS) is REQUIRED.
+# Enabling discovery with no advertise endpoint aborts boot (ErrNoEndpoint).
+# Canonical SD_* names below are read by libsd.ConfigFromEnv(). Legacy names
+# SD_ADVERTISE_* / SERVICE_ADVERTISE_* remain honored, so existing deployments
+# keep working without any change.
+SD_ENABLED=false
+# Consul HTTPS API (host:port); central agents typically listen on :8501.
+SD_ADDRESS=localhost:8500
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The SD_ADDRESS key should go before the SD_ENABLED key

(UnorderedKey)